src/Shell/BashShell.php

```diff
@@ -12,7 +12,7 @@ final class BashShell implements Shell
      * @param string $format
      * @param array $parameters
      *
-     * @return array
+     * @return string
```

Comment:
This is indeed a mistake. `array` is the correct type
https://www.php.net/manual/en/function.exec.php
Fixed the interface's type in https://github.com/asyncphp/doorman/pull/34